lib/src/Uncertainty/Distribution/MarshallOlkinCopula.cxx

```diff
@@ -0,0 +1,247 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The MarshallOlkinCopula distribution
+ *
+ *  Copyright 2005-2019 Airbus-EDF-IMACS-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include <cmath>
+#include "openturns/MarshallOlkinCopula.hxx"
+#include "openturns/RandomGenerator.hxx"
+#include "openturns/PersistentObjectFactory.hxx"
+#include "openturns/Exception.hxx"
+#include "openturns/SpecFunc.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+CLASSNAMEINIT(MarshallOlkinCopula)
+
+static const Factory<MarshallOlkinCopula> Factory_MarshallOlkinCopula;
+
+/* Default constructor */
+MarshallOlkinCopula::MarshallOlkinCopula()
+  : CopulaImplementation()
+  , alpha_(0.)
```

Comment:
Don't use 0 as a default value for alpha and beta as it is an invalid value for both arguments according to setAlpha() and setBeta(). A value of 0.5 is probably better.

Reply:
Ok. Fixed.